### ISSUE TITLE
Big Love Rocket doesn't upgrade Swift Flight Form to 310%

### ISSUE THREAD

**Issue by punims** [dalaranwow/dalaran-wow#5610]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Big Love Rocket a 310% flying mount doesn't cause druid Swift Flight Form to go 310%

**Expected behaviour**: Tell us what should happen instead
http://wowwiki.wikia.com/wiki/Patch_3.2.0 
According to patch notes of 3.2.0 
Swift Flight Form: Druids who have acquired a mount able to go 310% flight speed will now also go that speed while in this form.

As a side note Big Love Rocket was added in wotlk. I've heard a rumor that TBC 310% mounts (ashes)
didn't cause the speed increase but I haven't found proof of that

**Steps to reproduce the problem**:

1. Mount a 310% mount with one character
2. get a druid with a Big Love Rocket and use Swift Flight Form
3. have both characters on mounts, Druid in swift flight form, the other on any 310% mount 
and watch the druid not being able to keep up

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
Proof for Ashes  not giving speed - https://www.wowhead.com/item=32458/ashes-of-alar#comments:id=1365414
